pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -55,6 +56,7 @@ var (
 // managing Monitoring, Logging and Alerting for user clusters.
 // * project controller - create/update/delete Grafana organizations based on Kubermatic Projects
 // * userprojectbinding controller - create/update/delete Grafana Users to organizations based on Kubermatic UserProjectBindings
+// * cluster controller - create/update/delete Grafana Datasources to organizations based on Kubermatic Clusters
```

Comment:
Please update the docs, because those controller names have already changed. Also please update the doc.go

Reply:
done, thx